### PR TITLE
[Wallet] Add spinner, timer, and tip text to Verification input screen

### LINKED ISSUE #1523: Verification V2 (Optional Verification)
### Expected Behavior

Full Spec: https://docs.google.com/document/d/15oOHJlJk-yk19JRMZLO-Wm-iEB6tfgW4XcTtnNsJuEk/edit?pli=1#

Figma for onboarding: https://www.figma.com/file/9lBnIzsABhO8oXE0d0c8Km/Onboarding-and-Verification?node-id=10%3A557

Figma for tx flow: https://www.figma.com/file/yipmtvjgUreKhZyuK2slFg/Transaction-Flow%3A-Verification-and-Contact-Permissioning?node-id=0%3A1

Includes the Following (feel free to split up):
- invite code optional
- new verification flow
- post-onboarding prompts to re-verify

### Current Behavior

What is the current behavior?

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Add spinner, timer, and tip text to Verification input screen
- Improve code organization of related components

### Tested

Did verification and added new tests

### Related issues

- Fixes #1523 

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/verify/VerificationCodeRow.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    <View style={styles.codeInputContainer}>
+      <CodeInput
+        value={inputValue}
+        placeholder={CODE_INPUT_PLACEHOLDER}
+        shouldShowClipboard={shouldShowClipboard(attestationCodes)}
+        onChangeText={onInputChange}
+        style={styles.codeInput}
+      />
+      {isCodeSubmitting && (
+        <ActivityIndicator size="small" color={colors.celoGreen} style={styles.codeInputSpinner} />
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationCodeRow.tsx:77]: Don't we want our custom activity indicator everywhere now?

File: packages/mobile/src/verify/VerificationInputScreen.tsx [hunk 2]
```diff
@@ -168,6 +98,8 @@ class VerificationInputScreen extends React.Component<Props, State> {
       }
       this.setState({ timer: timer - 1 })
     }, 1000)
+    this.keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', this.onKeyboardShow)
+    this.keyboardDidHideListener = Keyboard.addListener('keyboardDidHide', this.onKeyboardHide)
```
Comment (jeanregisser) [inline packages/mobile/src/verify/VerificationInputScreen.tsx:102]: If you end up needing `<KeyboardSpacer />` on this screen for iOS, it has a `onToggle` prop that could simplify this a little.

**Review by jeanregisser — APPROVED** [review]
Looks great! 👍

File: packages/mobile/src/verify/VerificationCodeRow.tsx (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationCodeRow.tsx:77]: Not yet but soon!

File: packages/mobile/src/verify/VerificationInputScreen.tsx (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/verify/VerificationInputScreen.tsx:102]: Cool, I'll test iOS today and see if that's needed/helpful!

**Pull request merged**